server/tests/test_parameterval_views.py

```diff
@@ -37,8 +37,17 @@ def test_parameterval_detail_get(self):
         self.assertEqual(str_val['parameter_spec']['name'], 'StringParam')
         self.assertEqual(str_val['parameter_spec']['id_name'], 'stringparam')
         self.assertEqual(str_val['parameter_spec']['type'], ParameterSpec.STRING)
+        self.assertEqual(str_val['parameter_spec']['placeholder'], 'placeholder')
         self.assertEqual(str_val['value'], 'fooval')
 
+        # parameters have correct types and values
```

Comment:
Why the duplicated comment? It was introducing the block of parameter tests -- either remove or make it specific to this test you added, if you think capturing your intent here is useful.